packages/web/src/pages/explore-page/components/desktop/PlayableTile.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useCallback, useMemo } from 'react'
+
+import { useToggleTrack } from '@audius/common/hooks'
+import { ID, Kind, UID } from '@audius/common/models'
+import { QueueSource } from '@audius/common/store'
+import { makeUid } from '@audius/common/utils'
+
+import { TrackTileSize } from 'components/track/types'
+
+// Wrapper component to make tiles playable
+export const PlayableTile: React.FC<{
```

Comment:
generally dont use React.FC, just type the props

Reply:
Oh thanks, this was copied code